huggle/hugglequeuefilter.hpp

```diff
@@ -113,6 +119,9 @@ namespace Huggle
             HuggleQueueFilterMatch Self;
             HuggleQueueFilterMatch UserSpace;
             HuggleQueueFilterMatch TalkPage;
+            QString Type;
```

Comment:
The variables are set in the DefaultConfig
We could allow filtering for one of (or all of) a set of categories if you want